### ISSUE TITLE
Ability to select exposed input

### ISSUE THREAD

**Issue by dgarijo** [mintproject/mic#118]
Right now MIC will consider all inputs (or a folder with all inputs) thus not allowing me to expose inputs that I want a user to change in a component. I would like users to select which inputs to expose

**Comment by sirspock** [comment]
We cannot _ignore_ inputs because we need a link to download
I think that the problem is:
When I had a directory with several number files but I want to expose a small number. How I can do it?

**Comment by dgarijo** [comment]
Yes, the problem is how to expose a subset of the inputs that are used, even if underneath the component uses all of them. I guess that an additional thing to think about is should some of the `static` files be included in the container? This happened in Cycles, were we have template config files that are bundled with the invocation script.
